### DISCUSSION TITLE
Serialização de um XML de terceiros

### DISCUSSION THREAD

**Discussion by D1Alexandre** [TadaSoftware/PyNFe#232] [Q&A] [answered]
Tenho uma rotina no meu sistema onde preciso ler um XML de terceiros para capturar alguns dados, gostaria de saber se tem algum exemplo dentro do PyNFe

**Comment by leogregianin** [comment, marked as answer]
@D1Alexandre, na lib PyNFe não tem rotina para ler o XML.

Eu utilizo e sugiro utilizar a lib https://github.com/martinblech/xmltodict para isso, transforma o xml em dict python e fica muito fácil de trabalhar. Um exemplo de como utilizar aqui: https://github.com/marcelomaia/EasyGestor/blob/d10ef3b1c2a782d7ce29c1b88bb7f8903ad38269/stoqlib/lib/nfeimporter.py

Se quiser contribuir para o PyNFe e fazer essa leitura será bem vinda.
